tests/test_shuffle.py

```diff
@@ -1,12 +1,61 @@
+import uuid
+
 import dask
+import dask.dataframe as dd
+import pytest
+
+
+@pytest.fixture(scope="session")
+def s3_stability_url(s3, s3_bucket):
+    # Unique, because multiple tests are accessing the bucket
+    stability_url = f"{s3_bucket}/stability-scratch-{uuid.uuid4().hex}"
+
+    try:
+        s3.makedirs(stability_url)
+        yield stability_url
+
+    finally:
```

Comment:
I see that couple of the folders weren't deleted on S3 I wonder if it's from previous tries, or if there is something not entirely working. This are still in S3
![Screen Shot 2022-04-19 at 11 12 30 AM](https://user-images.githubusercontent.com/7526622/164037376-385b5337-417b-48b6-83ee-ce8a55e6d090.png)

Reply:
I think this is because a new commit was pushed up and the test run was forcibly cancelled by github actions. @jrbourbeau would you know how to handle this by any chance?

In any case, I think it's worth making all stability run data happens within a single root folder (but with unique sub-folders). I'll implement this change.